src/renderer/terms.js

```diff
@@ -31,38 +24,26 @@ export function acceptLendingTerms() {
   return global.localStorage.setItem("acceptedLendingTermsVersion", currentLendingTermsRequired);
 }
 
-export async function load(language: string) {
-  try {
-    const { data } = await network({ url: getRawLanguageURL(language) });
-    return data;
-  } catch (error) {
-    if (error.status === 404) {
-      const { data } = await network({ url: rawDefaultURL });
-      return data;
-    }
-    throw error;
-  }
-}
-
-export const useTerms = () => {
-  const [terms, setTerms] = useState(null);
-  const [error, setError] = useState(null);
+export const usePrivacyUrl = () => {
+  const [privacyUrl, setPrivacyUrl] = useState(urls.privacyPolicy.en);
 
   const language = useSelector(languageSelector);
   useEffect(() => {
-    load(language).then(setTerms, setError);
+    setPrivacyUrl(urls.privacyPolicy[language] || urls.privacyPolicy.en);
   }, [language]);
 
-  return [terms, error];
+  return privacyUrl;
 };
 
-export const usePrivacyUrl = () => {
-  const [privacyUrl, setPrivacyUrl] = useState(urls.privacyPolicy.en);
+// TODO: Merge useTermsUrl and usePrivacyUrl to create generic useDynamicUrl
```

Comment:
Will be happy to take few seconds to do that.
I'm thinking about developing a generic hook (`useDynamicUrl`) that accepts a translationKey param and return the correct URL if found. It will replace `useTermsUrl` and `usePrivacyUrl` which are redundant. 
Tell me if I have to include this modification to this pull request 👌

Reply:
Sure go for it. just make sure to search on the app all the instances wher this is called :+1: